lumicks/pylake/correlated_stack.py

```diff
@@ -184,6 +186,8 @@ def plot_correlated(self, channel_slice, frame=0, reduce=np.mean, channel="rgb")
         channel : 'rgb', 'red', 'green', 'blue', None; optional
             Channel to plot for RGB images (None defaults to 'rgb')
             Not used for grayscale images
+        figure_scale : float
+            Scaling of the plot.
```

Comment:
It's not super clear to me what this actually is. Is it the aspect ratio of the entire figure? Or the split ratio between the two plots? Might need a better docstring, maybe even a different parameter name

Reply:
Rephrased docstring to:
```
        figure_scale : float
            Scaling of the figure width and height. Values greater than one increase the size of the
            figure.
```